src/extension.ts

```diff
@@ -188,13 +188,14 @@ async function isSubscriptionManagerInstalled(): Promise<boolean> {
 }
 
 async function installSubscriptionManger() {
-  const exitCode = await runRpmInstallSubscriptionManager();
-  if (exitCode === undefined) {
-    throw new Error(
-      'Error running podman to install subscription-manager, please make sure podman machine is running!',
-    );
+  try {
+    const exitCode = await runRpmInstallSubscriptionManager();
+  } catch (err) {
+    const exitCode = (err as extensionApi.RunError).exitCode;
+    console.error(`Subscription manager installation returned exit code: ${exitCode}`);
```

Comment:
Tiny nit: As a user, I'd be more interested in the error message than the exit code.